lib/src/mistralai_client_dart_base.dart

```diff
@@ -19,8 +24,38 @@ class MistralAIClient {
   final int maxRetries;
   final http.Client _client;
 
+  Future<String> _getRequest<T>(String endpoint) async {
+    final headers = <String, String>{
+      'Accept': 'application/json',
+      'Content-Type': 'application/json',
+      'Authorization': 'Bearer $apiKey',
+    };
+
+    try {
+      final response = await _client
+          .get(Uri.parse('$baseUrl$endpoint'), headers: headers)
+          .timeout(timeout);
+
+      if (response.statusCode != 200) {
+        throw Exception('Get request failed: ${response.statusCode}');
+      }
+
+      return response.body;
+    } catch (e) {
+      throw Exception('Get request failed: $e');
+    }
+  }
+
+  /// Returns a list of the available models [ModelsList] 
+  /// from the mistral AI API.
```

Comment:
It looks like you have line length set to 80 and I think we should use at least 120 nowadays 😉

Reply:
That's not correct for `dart` @KacperKluka. Dart standard formatting is 80 line-length and you will lose score on [pub.dev](pub.dev) if it's not formatted correctly with `dart format`.

Reply:
Oh, good to know 😅 Thanks for the info, I'm closing the conversation then 👌